arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -277,7 +277,8 @@ public List<Module> getModulesToTrace() {
         this.shf,
         this.stp,
         this.txnData,
-        this.wcp);
+        this.wcp,
+        this.modexp.data());
```

Comment:
name of the module should be modexpData

Reply:
It's a submodule of `modexp`, which handles all the pre-processing.

Reply:
Yes, I wanted to eliminate the stutter.

Reply:
well, modexp is just a traceless module used to count the nb of modexp call. ModexData is a real module, triggered by modexp. We don't name `logInfo` and `logData` `rlpRcpt.info` or `rlpRcpt.data`

Reply:
It is named accordingly, this is just a getter from its `*EffectiveCall` parent.